src/sim-host/socket.js

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
That works too! It is even more clear to separate what is UI-related and what is not, and pass the path to the UI and not everything that can impact too to the protocol.
Thanks!